ledger-core/v2/vanilla/longbits/bytestring.go

```diff
@@ -210,3 +212,16 @@ func (v ByteString) FoldToBits64() (folded Bits64) {
 func (v ByteString) String() string {
 	return bitsToStringDefault(&v)
 }
+
+func (v ByteString) NewIoReader() io.Reader {
+	return strings.NewReader(string(v))
+}
+
+func (v ByteString) Hex() string {
```

Comment:
It is my understanding that this method doesn't has unit tests, although it's very easy to implement them. https://codecov.io/gh/insolar/assured-ledger/pull/94/changes Please add these tests.

Reply:
Well, and this is the answer - it is very easy to test because it does nothing worth of testing. 

Only creates a false feeling of good test coverage when all simple methods are tested, and complex ones are untested. Imho simple methods are the last to be covered.